boards/arm/ti_am62x_phyboard_m4/doc/index.rst

```diff
@@ -0,0 +1,149 @@
+
+phyBOARD-Lyra AM62x M4F Core
+############################
+
+Overview
+********
+
+The phyBOARD-Lyra AM62x board configuration is used by Zephyr applications
+that run on the TI AM62x platform. The board configuration provides support
+for the ARM Cortex-M4F MCU core and the following features:
+
+- Nested Vector Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+
+The board configuration also enables support for the semihosting debugging console.
+
+See the `PHYTEC AM62x Product Page`_ for details.
+
+.. figure:: images/phyCORE-AM62x_Lyra_frontside.webp
+   :align: center
+   :alt: phyBOARD-Lyra AM62x
+
+   PHYTEC phyBOARD-Lyra with the phyCORE-AM62x SoM
+
+Hardware
+********
+The phyBOARD-Lyra AM62x kit features the AM62x SoC, which is composed of a
+quad Cortex-A53 cluster and a single Cortex-M4 core in the MCU domain. Zephyr
+is ported to run on the M4F core and the following listed hardware
+specifications are used:
+
+- Low-power ARM Cortex-M4F
+- Memory
+
+   - 256KB of SRAM
+   - 2GB of DDR4
+
+- Debug
+
+   - XDS110 based JTAG
+
+Supported Features
+==================
+
+The phyboard_lyra_am62x_m4 configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINCTRL   | on-chip    | pinctrl                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial                              |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by the port.
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 400 MHz.
+
+DDR RAM
+-------
+
+The board has 2GB of DDR RAM available. This board configuration
+allocates Zephyr 4kB of RAM (only for resource table: 0x9CC00000 to 0x9CC00400).
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+MCU domain UART (MCU_UART0).
+
+SD Card
+*******
+
+Download PHYTEC's official `WIC`_ as well as `BMAP`_ and flash the WIC file with
+an etching software onto an SD-card. This will boot Linux on the A53 application
+cores of the SoM. These cores will then load the zephyr binary on the M4 core
+using remoteproc.
+
+The default configuration can be found in the defconfig file:
+
+.. code-block:: console
+
+   boards/arm/phyboard_lyra_am62x_m4/phyboard_lyra_am62x_m4_defconfig
+
+Flashing
+********
+
+The board can using remoteproc, and uses the OpenAMP resource table to accomplish this.
+
```

Comment:
nitpick: "can be loaded" , also instead of board it would be better to mention M4 core.